dss-signature-policy-validator/src/main/java/eu/europa/esig/dss/signature/policy/validation/FullCAdESSignaturePolicyValidator.java

```diff
@@ -364,4 +374,12 @@ private CAdESSignature getCadesSignature() {
 	private boolean isCadesSignature() {
 		return getSignature() instanceof CAdESSignature;
 	}
+	
+	protected void addErrors(String key, Object... extraParams) {
```

Comment:
Shouldn't this method name be on singular?

Reply:
It shall be. Initially it wasn't because of incorrect param overload, but I changed the method and forgot to rename it.